meeting-notes/2019-Jun-06.md

```diff
@@ -0,0 +1,46 @@
+# Falco Office hours meeting 6th June 2019
+
+**MC:** Lorenzo Fontana
+
+**Who joined**:
+
+- Lorenzo Fontana
+- Mark Stemm
+- Michael Ducy
+- Leonardo Di Donato
+- Spencer Krum
+- Thomas Labarussias
+- Mattia Lavacca
+
+This was the first official Office hours meeting! We had two proposals to discuss
+and there have been 7 people participating very actively.
+
+## Proposals
+
+[#5](https://github.com/falcosecurity/office-hours/issues/5): Improved Falco outputs
+
+Thomas Labarussias presented it as proposed in the issue, slightly different than what was proposed initially on  having a grpc output.
+
+- **Ducy:** thinks that it should be only a  grpc plugin and not a sidekick 
+- **Leo:** Also if I’d like to have fan-in/out mechanisms and broadcasting of events, I think the architecture is a bit overkill, for example I have concerns about all the queues in the diagram and the different mechanisms used to grab data from them (push vs pull)
+- Mark and Kaizhe think that the architecture is a bit giant, needs to be simpler, the diagram
+- **Kaizhe Question:** when I look into the diagram I can’t understand what Falco is expected to provide, autiding events or what? It’s not that crazy now, trying to reduce noise, what do you expect from Falco regarding events.
+- **Thomas Answer:** I just want a very simple configuration on which plugins are enabled and what events are received, if you take my program I didn’t integrate the idea of minimum criticity, it’s only that, which plugin are enabled and events. No rules on which kind of messages etc.. Falco has to be the truth, the base of knowledge.
+- **Kaizhe:** I’m more looking into this to enrich how this new architecutre can bring the detection capabilities to the next Level, I see some flexibility here when we have a broker and it can’t be easily enrich the Falco events but I think I still agree with Mark, maybe start with simple and see what we can do from there
```

Comment:
```suggestion
- **Kaizhe:** I'm more looking into this to enrich how this new architecture can bring the detection capabilities to the next level, I see some flexibility here when we have a broker and it can't be easily to enrich the Falco events but I think I still agree with Mark, maybe start with simple and see what we can do from there.
```